tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -58,16 +62,26 @@ module('Acceptance | guid file | registration files', hooks => {
         assert.dom('[data-test-tags-button]').exists('Tags button exists');
         await percySnapshot('Acceptance | guid file | registration files | mobile view | file renderer');
 
+        assert.dom('[data-test-file-renderer]').isVisible();
+        assert.dom('[data-test-metadata-tab]').isNotVisible();
```

Comment:
It appears that `data-test-metadata-tab` is not a test selector that I put in.

Reply:
Same here in reverse.